packages/svelte-flicking/src/FlickingPanel.js

```diff
@@ -0,0 +1,7 @@
+/*
```

Comment:
d.ts files are available in `flicking-panel.svelte.d.ts` format.

Reply:
I'll try that 👍